### PR TITLE
Use the mimetype when deciding the transformer

### PR DESCRIPTION
**Pull request by morty**

### CODE REVIEW

File: ckanext/datapreview/transform/base.py [hunk 1]
```diff
@@ -12,9 +12,9 @@ def find_transformer(extension=None, mime_type=None):
 
     info = None
     for trans in transformers:
-        if extension and extension in trans["extensions"]:
+        if extension and extension.lower() in trans["extensions"]:
             info = trans
-        if mime_type and mime_type in trans["mime_types"]:
+        if mime_type and mime_type.lower().startswith(trans["mime_types"]):
```
Comment (davidread) [inline ckanext/datapreview/transform/base.py]: I don't think this is right - you can't do startswith() with a list of strings

Comment (morty) [inline ckanext/datapreview/transform/base.py]: You're right, it cannot take a list of strings. It's a tuple of strings. https://docs.python.org/2/library/stdtypes.html#str.startswith

Comment (davidread) [inline ckanext/datapreview/transform/base.py]: Ok, I learnt something!

**Comment by davidread** [comment]
Can you tell me how I can test this?

**Comment by davidread** [comment]
Name an example resource or something

**Pull request merged**